PocketKit/Sources/PocketKit/Collections/CollectionViewModel.swift

```diff
@@ -5,21 +5,37 @@
 import Sync
 import UIKit
 import SharedPocketKit
+import Combine
 
 // View model that holds logic for the native collection view
 class CollectionViewModel {
+    typealias EventPublisher = AnyPublisher<ReadableEvent, Never>
     typealias Snapshot = NSDiffableDataSourceSnapshot<Section, Cell>
 
     @Published var snapshot: Snapshot
 
+    private var _events = PassthroughSubject<ReadableEvent, Never>()
+    var events: EventPublisher { _events.eraseToAnyPublisher() }
+
+    @Published private(set) var _actions: [ItemAction] = []
+    var actions: Published<[ItemAction]>.Publisher { $_actions }
```

Comment:
Any specific reason why we are using two different patterns for `events` and `actions`?

Reply:
updated to: 
```
 @Published private(set) var _events: ReadableEvent?
    var events: Published<ReadableEvent?>.Publisher { $_events }
```